dashboard/src/components/Config/PkgRepoList/PkgRepoForm.tsx

```diff
@@ -320,7 +321,7 @@ export function PkgRepoForm(props: IPkgRepoFormProps) {
         finalHeader = authCustomHeader;
         break;
       case PackageRepositoryAuth_PackageRepositoryAuthType.PACKAGE_REPOSITORY_AUTH_TYPE_BEARER:
-        finalHeader = `Bearer ${bearerToken}`;
+        finalHeader = bearerToken;
```

Comment:
This change is surprising? Was it a bug currently, or why is the `Bearer ` not included now?

Reply:
i think this was a left-over bug from when the UI was switched to the new API.
The UI code was not changed to comply with what the spec says. The UI was sending the token prefixed with Bearer, and the backend would strip the prefix if present, only (for helm) to add it back before putting it in the secret.
On a read, the backend would always send the token only, without the Bearer prefix (as per the spec). So the UI was inconsistent between its read and write operations.
this is double defensive and inconsistent code that makes things so hard to read and understand, so i cleaned it up.